src/components/vehicleReference/index.tsx

```diff
@@ -52,3 +52,4 @@ const VehicleReference: FC<Props> = ({
 };
 
 export default VehicleReference;
+export { Props as VehicleProps };
```

Comment:
```suggestion
export { Props as VehicleReferenceProps };
```
(to keep consistency with other prop exports)